.gitlab-ci.yml

```diff
@@ -52,8 +52,11 @@ Testing:
           - aws/fedora-33-x86_64
           - aws/fedora-33-aarch64
           - aws/rhel-8.4-ga-x86_64
+          - aws/rhel-8.4-ga-aarch64
           - aws/rhel-8.6-nightly-x86_64
+          - aws/rhel-8.6-nightly-aarch64
           - aws/rhel-9.0-beta-nightly-x86_64
+          - aws/rhel-9.0-beta-nightly-aarch64
```

Comment:
Shouldn't these be rhel-9.0-nightly instead of beta ?

Reply:
Yes, I planned to update them after they got merged in osbuild-composer repository. However there is currently an issue running the tests on rhel-9 https://github.com/osbuild/osbuild-composer/issues/1819 which also relates to more issues.. So I guess I'll have to wait for it to get resolved otherwise I can't switch... At least the related PR in osbuild-composer repository seems to be doing what I intended and the openstack tests on aarch64 pass now with just the boot part being skipped.

Reply:
Q: Is the current issue on RHEL 9 blocking all tests on aarch64 or only some of them ?

Reply:
>  Q: Is the current issue on RHEL 9 blocking all tests on aarch64 or only some of them ?

Well since rhel-9-beta runners are no longer being used and we're currently not testing on rhel-9 because of the above mentioned issue anyway I could just rebase and it should be good to go. Once the issue is fixed we'll just uncomment the tests on rhel-9...